test/s1/appframework/appframework_test.go

```diff
@@ -448,6 +453,12 @@ var _ = Describe("s1appfw test", func() {
 			// Verify Monitoring Console is Ready and stays in ready state
 			testenv.VerifyMonitoringConsoleReady(deployment, deployment.GetName(), mc, testenvInstance)
 
+			// Upload apps to S3 for Standalone
```

Comment:
Nit: Comment should be both for standalone and MC

Reply:
Uploading for standalone here. MC upload is at the beginning of the test.